modules/web-console/frontend/app/types/index.ts

```diff
@@ -65,5 +65,6 @@ export type NavigationMenu = Array<NavigationMenuItem>;
 
 export enum WellKnownOperationStatus {
     WAITING = 'WAITING',
-    ERROR = 'ERROR'
+    ERROR = 'ERROR',
+    ARRIVED = 'ARRIVED'
```

Comment:
`ARRIVED` and `OperationStatus` don't go well together. Maybe `DONE` or `COMPLETE`?

Reply:
Agree. Changed to `DONE`